idurar/idurar-erp-crm
# Fixed invalid dashboard due balance amount

## Description
The due balance amount was previously considering only unpaid invoices.
It no considers partially paid invoices too.
Total due balance = (total of unpaid invoices) + [total of balance amount (invoice total - credit) on partially paid invoices]

## Related Issues
https://github.com/idurar/idurar-erp-crm/issues/786

## Steps to Test
1. Go to the application.
2. Create an invoice (example - $100).
3. Register a partial payment (example - $30; current due = $100 - $30 = $70) against the invoice you created in above step. 
4. Validate the due balance shown on the dashboard (it should be $70 as long as there is no other invoice on the system).

## Checklist
